util/memory/tracker.go

```diff
@@ -654,6 +654,32 @@ func (t *Tracker) setParent(parent *Tracker) {
 	t.parMu.parent = parent
 }
 
+func (t *Tracker) CountAllChildrenMemUse() map[string]int64 {
+	trackerMemUseMap := make(map[string]int64, 1024)
+	countChildMem(t, "", trackerMemUseMap)
+	return trackerMemUseMap
+}
+
+func countChildMem(t *Tracker, familyTreeName string, trackerMemUseMap map[string]int64) {
+	familyTreeName += "[" + strconv.Itoa(t.Label()) + "] <- "
+	if _, ok := trackerMemUseMap[familyTreeName]; !ok {
+		trackerMemUseMap[familyTreeName] = t.BytesConsumed()
+	} else {
+		trackerMemUseMap[familyTreeName] += t.BytesConsumed()
+	}
+	if len(t.mu.children) == 0 {
+		return
+	}
```

Comment:
Why this check is not in t.mu.Lock() scope?

Reply:
done.